### PR TITLE
ci(travis): update to Xenial

### PR DESCRIPTION
**Pull request by douglasduteil**
---

Following https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment

### CODE REVIEW

**Comment by lionelB** [comment]
can you elaborate a bit on these changes (why)

**Comment by douglasduteil** [comment]
\from https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment
> Xenial as the default Travis CI build environment is coming up next week, starting incrementally on Tuesday April 23rd, 2019.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-dist: trusty
-sudo: false
-language: generic
+dist: xenial
```
Comment (lionelB) [inline .travis.yml:1]: I wonder if we could just  remove the `dist`, since we always use the default value ?

Comment (douglasduteil) [reply .travis.yml:1]: :thinking: Let me try

**Review by lionelB — APPROVED** [review]

**Comment by douglasduteil** [comment]
Naaaa it's still "trusty" by default :(
see 
https://travis-ci.com/SocialGouv/code-du-travail-numerique/jobs/201812728#L9
vs https://travis-ci.com/SocialGouv/code-du-travail-numerique/builds/112481157#L9

**Pull request merged**